datetimerange/__init__.py

```diff
@@ -106,48 +114,57 @@ def __isub__(self, other):
 
         return self
 
-    def __contains__(self, x):
+    def __contains__(self, other):
         """
-        :param x:
+        :param other:
             |datetime|/``DateTimeRange`` instance to compare.
             Parse and convert to |datetime| if the value type is |str|.
-        :type x: |datetime|/``DateTimeRange``/|str|
-        :return: |True| if the ``x`` is within the time range
+        :type other: |datetime|/``DateTimeRange``/|str|
+        :return: |True| if the ``other`` is within the time range
         :rtype: bool
-
         :Sample Code:
             .. code:: python
-
                 from datetimerange import DateTimeRange
-
                 time_range = DateTimeRange("2015-03-22T10:00:00+0900", "2015-03-22T10:10:00+0900")
                 print("2015-03-22T10:05:00+0900" in time_range)
                 print("2015-03-22T10:15:00+0900" in time_range)
-
                 time_range_smaller = DateTimeRange("2015-03-22T10:03:00+0900", "2015-03-22T10:07:00+0900")
                 print(time_range_smaller in time_range)
         :Output:
             .. parsed-literal::
-
                 True
                 False
                 True
-
```

Comment:
Why these lines are removed from docstring?

Reply:
Maybe this was my IDE